### ISSUE TITLE
[HOLD for payment 2024-01-17] [$500] Grey box is shown instead of bank icon

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#33664]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.18-1
**Reproducible in staging?:** y
**Reproducible in production?:** not able to check
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @shubham1206agra 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1703677023896169

## Action Performed:
1. Go to Workspace > Bank Account
2. Select Connect online via plaid
3. Select any bank, and complete the plaid process
4. Observe the icon next to bank name

## Expected Result: 
Bank Icon should be shown

## Actual Result: 
Grey Box is shown

## Workaround:
unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

![Screenshot_2023-12-26-23-04-55-360_com expensify chat adhoc](https://github.com/Expensify/App/assets/38435837/1808fd4f-c274-495d-8d6d-2c99bdbc7995)

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~018bcfe6522be3b9f5</li>
        <li>Upwork Job ID: 1740403705181048832</li>
        <li>Last Price Increase: 2023-12-28</li>
<li>Automatic offers: </li>
<ul>
<li>shubham1206agra | Contributor | 28071698</li>
<li>Pujan92 | Contributor | 28071779</li>
</ul></li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @slafortune</details>

**Comment by Pujan92** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Bank icon filled with grey color

### What is the root cause of that problem?
In this [PR](https://github.com/Expensify/App/pull/30905), for many places we have added fill property to Illustrations or images. In some screenshots just showing the image issue in diff component.

- ImageCropView
- AddPlaidBankAccount
- DeeplinkRedirectLoadingIndicator
- ValidateCodeModal
- LogInWithShortLivedAuthTokenPage

![Simulator Screen Shot - iPhone 14 Pro Max - 2023-12-28 at 03 20 57](https://github.com/Expensify/App/assets/14358475/f2fc46f6-5e68-46c1-bc9e-27d075b54286)
![Simulator Screen Shot - iPhone 14 Pro Max - 2023-12-28 at 03 26 49](https://github.com/Expensify/App/assets/14358475/6a1138ad-9d09-4533-b6ee-842ece15e37c)
![Simulator Screen Shot - iPhone 14 Pro Max - 2023-12-28 at 03 34 11](https://github.com/Expensify/App/assets/14358475/04c8e5e4-5f69-444e-9a71-dc81d959c8e3)

### What changes do you think we should make in order to solve the problem?
Remove `fill` property for Illustrations or images from all these components.

**Comment by shubham1206agra** [comment]
Sorry everyone, but I am working on this already actually. I just reported this for GH issue

**Comment by mountiny** [comment]
this indeed has already been worked on by @shubham1206agra 

This is not a blocker since this is already in production

Taking over since I was involved in the similar fixes @srikarparsi if thats ok

**Comment by shubham1206agra** [comment]
@mountiny can we hire @Pujan92 for partial compensation (he helped me to find extra places where icon has problems)?, and let's fix all places at once

**Comment by mountiny** [comment]
@shubham1206agra do you want to let them fix it and you will be c+?

**Comment by shubham1206agra** [comment]
I will be fine as C+. And I will finish that PR and @Pujan92 open another PR to fix remaining places

**Comment by mountiny** [comment]
I just dont want to do extra payments given we would need another C+ for this

**Comment by Pujan92** [comment]
I think in the same PR we can update other occurrences.

**Comment by situchan** [comment]
Why not in same PR as they have the same root cause?
As said in https://github.com/Expensify/App/issues/33664#issuecomment-1870761546, this GH was just created for tracking purpose. No need separate fix

**Comment by mountiny** [comment]
Yep we should make one PR @shubham1206agra

**Comment by shubham1206agra** [comment]
> @shubham1206agra do you want to let them fix it and you will be c+?

Lets do this in single PR then, and I will be C+ here then :)

**Comment by shubham1206agra** [comment]
@mountiny Can you hire @Pujan92 in this case then?

**Comment by situchan** [comment]
In this case, I think @Pujan92 can do reviewer role. Code is simple and just let them double test changes.
And full compensation to him.

**Comment by mountiny** [comment]
If you have the PR, then raise it @shubham1206agra if not, than lets ask @Pujan92 to make the PR

**Comment by shubham1206agra** [comment]
I do have the PR already. I just need to update the remaining places

**Comment by mountiny** [comment]
@Pujan92 Can you do C+ role in this case then?

**Comment by Pujan92** [comment]
Yes @vitHoracek , I will test and share update here once PR is raised.

**Comment by mountiny** [comment]
@Pujan92 @shubham1206agra Whats the latest on this one, could we focus on getting this PR merged?

**Comment by shubham1206agra** [comment]
We are stuck on ImageCropView. Requires your input there.

**Comment by shubham1206agra** [comment]
@dannymcclain You need to comment on https://github.com/Expensify/App/pull/33607#issuecomment-1874054095.

**Comment by dannymcclain** [comment]
I'm sorry but I don't think I'm understanding what the issue is? cc @shawnborton in case you have more context on this (if not just ignore).

**Comment by shubham1206agra** [comment]
@dannymcclain You need to comment that which design is better in case of `ImageCropView`. The different view is in https://github.com/Expensify/App/pull/33607#issuecomment-1874054095

**Comment by shawnborton** [comment]
Ah, I had reported something similar [here](https://expensify.slack.com/archives/C049HHMV9SM/p1704218743166749)

But yeah, seems like the bank account icon went missing and we should be showing the default bank icon when we don't have a logo available to us. cc @grgia @madmax330 in case you have more context too

**Comment by mountiny** [comment]
@shawnborton i think thank bank icon issue should be handled already in the PR, but @shubham1206agra has this question they need help with https://github.com/Expensify/App/pull/33607

**Comment by shubham1206agra** [comment]
False alert above as PR is already in prod.

**Comment by slafortune** [comment]
@Pujan92 - Paid 👍 
@shubham1206agra can you complete the checklist above and also accept this [offer](https://www.upwork.com/nx/wm/offer/28071698)

**Comment by slafortune** [comment]
@shubham1206agra will you be able to complete the above checklist in the next day or two?

**Comment by shubham1206agra** [comment]
**BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:

* [x]  [@shubham1206agra] The PR that introduced the bug has been identified. Link to the PR: https://github.com/Expensify/App/pull/30905
* [x]  [@shubham1206agra] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment: https://github.com/Expensify/App/pull/30905/files#r1463416802
* [x]  [@shubham1206agra] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion: https://expensify.slack.com/archives/C049HHMV9SM/p1706022333430269
* [x]  [@shubham1206agra] Determine if we should create a regression test for this bug. No
* [x]  [@shubham1206agra] If we decide to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again.
* [ ]  [@slafortune] Link the GH issue for creating/updating the regression test once above steps have been agreed upon:

**Comment by slafortune** [comment]
Thanks @shubham1206agra Pd 👍
